src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -30,20 +30,11 @@ class DumpDataCollector extends DataCollector implements DataDumperInterface
     private $isCollected = true;
     private $clonesCount = 0;
     private $clonesIndex = 0;
-    private $rootRefs;
 
     public function __construct(Stopwatch $stopwatch = null, $fileLinkFormat = null)
     {
         $this->stopwatch = $stopwatch;
         $this->fileLinkFormat = $fileLinkFormat ?: ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
-
-        // All clones share these properties by reference:
-        $this->rootRefs = array(
```

Comment:
This is black magic with references: by holding references to other properties in the array, it turns these props to references. By doing so, all clones of a DumpDataCollector object share these props by reference. The comment was here to say that. Feel free to make it better :)